test/units/modules/network/ovs/test_openvswitch_bridge.py

```diff
@@ -155,6 +160,21 @@ def test_openvswitch_bridge_present_creates_fake_bridge(self):
         self.execute_module(changed=True, commands=commands,
                             test_name='test_openvswitch_bridge_present_creates_fake_bridge')
 
+    @patch.object(openvswitch_bridge, 'map_config_to_obj')
+    def test_openvswitch_bridge_updates_vlan(self, m_have):
+        m_have.return_value = {'bridge': 'test-br2', 'parent': 'test-br',
+                               'vlan': 200, 'fail_mode': None,
+                               'external_ids': None, 'set': None}
+        set_module_args(dict(state='present',
```

Comment:
We usually prefer dict literals `{}` over `dict()` constructors/factories. (Although it's a holywar topic)